frontend/device/kindle/device.lua

```diff
@@ -1318,6 +1319,23 @@ function KindleScribe:init()
 
     Kindle.init(self)
 
+    self.input:registerEventAdjustHook(ZeldaGyroTranslation)
+    self.input.handleMiscEv = function(this, ev)
+        if ev.code == C.MSC_GYRO then
+            return this:handleGyroEv(ev)
+        end
+    end
+
+    -- Get accelerometer device by looking for EV=9
+    local std_out = io.popen("grep -e 'Handlers\\|EV=' /proc/bus/input/devices | grep -B1 'EV=9' | grep -o 'event[0-9]\\{1,2\\}'", "r")
+    if std_out then
+        local rotation_dev = std_out:read("*line")
+        std_out:close()
+        if rotation_dev then
+            self.input.open("/dev/input/"..rotation_dev)
+        end
+    end
```

Comment:
This is probably overkill, check `/dev/input/by-path`, with a bit of luck the kx entry should have a symlink there.

Reply:
Yes, it is `/dev/input/by-path/platform-11007000.i2c-event-joystick`.

I will replace it.